kommunicateui/src/main/java/com/applozic/mobicomkit/uiwidgets/conversation/activity/FullScreenImageActivity.java

```diff
@@ -104,8 +123,16 @@ public void onResourceReady(@NonNull Bitmap resource, @Nullable Transition<? sup
 
             if (message != null && message.getFilePaths() != null && !message.getFilePaths().isEmpty()) {
                 try {
-                    Bitmap imageBitmap = ImageUtils.decodeSampledBitmapFromPath(message.getFilePaths().get(0));
-                    mediaImageView.setImageBitmap(imageBitmap);
+                    if(message.getFileMetas().getContentType().contains("gif")) {
+                        Glide.with(this)
+                                .asGif()
+                                .load(message.getFilePaths().get(0))
```

Comment:
same here get(0) will produce fatal error. Check the next comment

Reply:
We are already doing a check:
if (message != null && message.getFilePaths() != null && !message.getFilePaths().isEmpty())